manuscript/paper.tex

```diff
@@ -117,13 +117,32 @@ \section{Introduction}
 recombining
 species, and are currently the subject of intense research
 interest~\citep{brandt2024promise,lewanski2024era,nielsen2024inference,
-    wong2024general}. ARGs are a fundamental object in population genetics,
-and although they have been of theoretical interest for
-decades~\citep{Hudson1983,Griffiths1997} it is only with recent breakthroughs
-in inference methods~\citep{rasmussen2014genome,
-    speidel2019method,kelleher2019inferring,wohns2022unified,zhang2023biobank,
-    gunnarsson2024scalable,deng2024robust} that practical application has
```

Comment:
I think we can do something much smaller here - just change "practical" to "widespread"